db/seeds.rb

```diff
@@ -1,7 +1,61 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+User.create!(
+  first_name: 'Евгений',
+  last_name: 'Побережный',
+  email: 'mail@mail.com',
+  password: 'A1qwerty',
```

Comment:
Looks like this line contains sensitive data.

Reply:
fixed